cni/network/network_windows.go

```diff
@@ -111,7 +111,7 @@ func updateSubnetPrefix(cnsNwConfig *cns.GetNetworkContainerResponse, subnetPref
 
 func getNetworkName(podName, podNs, ifName string, nwCfg *cni.NetworkConfig) (string, error) {
 	if nwCfg.MultiTenancy {
-		_, cnsNetworkConfig, _, err := getContainerNetworkConfiguration(nwCfg, "", podName, podNs, ifName)
+		_, cnsNetworkConfig, _, err := getContainerNetworkConfiguration(nwCfg, podName, podNs, ifName)
```

Comment:
As discussed, if we don't handle an operation ona  given platform we should explicitly throw an error, operation is not support rather silently not doing anything.